### PR TITLE
Updating manifests to include Deployed By and Attributes

### PR DESCRIPTION
**Pull request by datafranch**

### CODE REVIEW

File: adapters/amazon_review_sentiment_analysis/dataloop.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "name": "amazon-review-sentiment-analysis",
   "description": "This model is a fine-tuned version of nlptown/bert-base-multilingual-uncased-sentiment on an Amazon US Customer Reviews Dataset.",
   "attributes": {
-    "Provider": "Hugging Face",
+    "Provider": "AWS",
+    "Deployed By": "Dataloop",
     "License": "Apache 2.0",
     "Category": "Model",
     "Media Type": "Text",
```
Comment (ShadiMah) [inline adapters/amazon_review_sentiment_analysis/dataloop.json:9]: Media type is a List (Verify for all)

Comment (ShadiMah) [inline adapters/amazon_review_sentiment_analysis/dataloop.json:9]: Add Gen AI attribute (Verify for all)

File: adapters/auto_for_causal_lm/dataloop.json [hunk 2]
```diff
@@ -3,10 +3,12 @@
   "description": "AutoForCausalLM for chatting - HF",
   "attributes": {
     "Provider": "Hugging Face",
```
Comment (ShadiMah) [inline adapters/auto_for_causal_lm/dataloop.json:5]: HF provider?

File: adapters/uform_gen2_qwen_500m/dataloop.json [hunk 3]
```diff
@@ -2,13 +2,15 @@
   "name": "uform-gen2-qwen-500m-huggingface-app",
   "description": "UForm-Gen is a small generative vision-language model primarily designed for Image Captioning and Visual Question Answering.",
   "attributes": {
-    "Provider": "Hugging Face",
+    "Provider": "Unum",
```
Comment (ShadiMah) [inline adapters/uform_gen2_qwen_500m/dataloop.json:5]: Who is Unum?

**Review by ShadiMah — CHANGES_REQUESTED** [review]

File: adapters/uform_gen2_qwen_500m/dataloop.json (hunk 3 above)
Comment (datafranch) [reply adapters/uform_gen2_qwen_500m/dataloop.json:5]: The provider for this model. I checked with Or, he told me to find the appropriate company that provided each one and add them and ask Liran to add the appropriate logos. Should we only do it if it's a bigger company?

File: adapters/auto_for_causal_lm/dataloop.json (hunk 2 above)
Comment (datafranch) [reply adapters/auto_for_causal_lm/dataloop.json:5]: In this case, yes, because it's a generic adapter using HF classes

**Review by orshabtay — APPROVED** [review]

**Pull request merged**